src/EventStore.Core/Services/Transport/Grpc/Enumerators.cs

```diff
@@ -1,4 +1,5 @@
 namespace EventStore.Core.Services.Transport.Grpc {
 	internal static partial class Enumerators {
+		private const int MaxLiveEventBufferCount = 16;
```

Comment:
Does this need to be configurable?

Reply:
I'm going to say no, except for possibly testing purposes

Reply:
This should be configurable, and 16 is likely too low as a default.
If I remember correctly, the default in the existing catchup subscriptions is in the hundreds

Reply:
I am reluctant to make this configurable as subscriptions now live on the server, setting this too high could take down a node.

Reply:
Fair enough